### ISSUE TITLE
[BUG] rancher-cli v2.7.0 errors out trying to create an imported cluster

### ISSUE THREAD

**Issue by bfbachmann** [rancher/rancher#41144]
**Rancher Server Setup**
- Rancher version: Server v2.7.1, CLI v2.7.0
- Installation option (Docker install/Helm Chart): Docker
- Proxy/Cert Details: Default self-signed

**Information about the Cluster**
- Kubernetes version: n/a
- Cluster Type (Local/Downstream): n/a
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider): n/a

**User Information**
- What is the role of the user logged in? Admin

**Describe the bug**
The Rancher CLI is no longer able to create imported clusters. When running `rancher cluster create --import my-new-cluster`, get an error saying its not found.

**To Reproduce**
With the Rancher CLI:
1. `rancher login --token <your api token> <your server URL>`
2. `rancher cluster create --import <your new cluster name>`

**Result**
Get error `FATA[0001] resource of type cluster with ID my-new-cluster: not found`

**Expected Result**
A new imported cluster is created.

**Screenshots**

**Additional context**
SURE-6085

**Comment by markusewalker** [comment]
**QA TEST PLAN**

Scenarios
|Scenario|Test Case|
|---|---|
|1|Using the Rancher CLI, attempt to create an imported cluster, validate that it creates in the CLI and Rancher|

**Comment by markusewalker** [comment]
Reproduced this on Rancher CLI `v2.7.0`. See below:

ENVIRONMENT DETAILS
- Rancher install: Docker
- Rancher version: `v2.7-head` https://github.com/rancher/rancher/commit/d3b5469238e965e54306f74d0e98a47b780cbe80
- Rancher CLI version: `v2.7.0`
- Browser: Chrome

**TEST RESULT**
|Scenario|Test Case|Result
|---|---|---|
|1|Using the Rancher CLI, attempt to create an imported cluster, validate that it creates in the CLI and Rancher|:x:|

**REPRO STEPS**

**Scenario 1**
1. In a client machine with Rancher CLI `v2.7.0` setup, ran command `rancher login --token <admin bearer token> <Rancher server URL>`.
2. Ran command `rancher cluster create --import test` and received the following error message:
```
$ rancher cluster create --import test
FATA[0000] Not found: test
```
I will attempt now to test on Rancher CLI `v2.7.6-rc1` to see if this addressed.

**Comment by markusewalker** [comment]
Validated that this is addressed on Rancher CLI `v2.7.6-rc1`. See below:

**ENVIRONMENT DETAILS**
- Rancher install: Docker
- Rancher version: `v2.7-head` https://github.com/rancher/rancher/commit/d3b5469238e965e54306f74d0e98a47b780cbe80
- Rancher CLI version: v2.7.6-rc1
- Browser: Chrome

**TEST RESULT**
|Scenario|Test Case|Result
|---|---|---|
|1|Using the Rancher CLI, attempt to create an imported cluster, validate that it creates in the CLI and Rancher|:white_check_mark:|

**VALIDATION STEPS**

**Scenario 1**
1. In a client machine with Rancher CLI `v2.7.6-rc1` setup, ran command `rancher login --token <admin bearer token> <Rancher server URL>`.
2. Ran command `rancher cluster create --import test` and received the following error message:
```
$ rancher cluster create --import test
Successfully created cluster test
```
3. Navigated to Rancher in a browser and validated that it is created:
![image](https://github.com/rancher/rancher/assets/52591819/517de812-887b-4b1a-88ad-0fd5c7279dcd)
4. In a separate client machine, stood up a K3S standalone cluster and applied the registration command for the created `test` cluster.
5. Validated that the cluster is `Active` in Rancher:
![image](https://github.com/rancher/rancher/assets/52591819/a081e605-9778-4e5d-917c-28e65380845d)
     - For transparency, this was a new `test` cluster as I deleted the old one. Hence why the age is shorter than the previous screen capture.
